File: app/src/main/java/com/example/dailyexpensemanager/viewModels/VM_EditDataExpense.java
package com.example.dailyexpensemanager.viewModels;

import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class VM_EditDataExpense extends ViewModel {
    public String category="",amount="",dateTime="";
    public String  time,day,month,year;


    public MutableLiveData<String > mutable_category = new MutableLiveData<>();
    public MutableLiveData<String> mutable_amount = new MutableLiveData<>();
}

File: app/src/main/java/com/lincoln4791/dailyexpensemanager/viewModels/VM_EditDataExpense.kt
package com.lincoln4791.dailyexpensemanager.viewModels

import androidx.lifecycle.ViewModel
import androidx.lifecycle.MutableLiveData
import dagger.hilt.android.AndroidEntryPoint


class VM_EditDataExpense : ViewModel() {
    var category = ""
    var amount = ""
    var dateTime = ""
    var time: String? = null
    var day: String? = null
    var month: String? = null
    var year: String? = null
    var mutable_category = MutableLiveData<String>()
    var mutable_amount = MutableLiveData<String>()
}